scripts/benchmark_bolt.py

```
@@ -0,0 +1,100 @@
+        start = time.perf_counter()
+        res = client.get("/")
+        end = time.perf_counter()
+        if res.status_code != 200:
+            print(f"Error: {res.status_code}")
+            # print(res.data.decode())
+        latencies.append(end - start)
+
+    avg = statistics.mean(latencies)
+    median = statistics.median(latencies)
+    p95 = statistics.quantiles(latencies, n=20)[18]
+    print(f"Benchmark results for 50 crew members:")
```

Ruff (0.15.7)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)